personal/korneliacwik/reports/zajecia3.Rmd

```diff
@@ -0,0 +1,87 @@
+---
+title: "Zajęcia 3"
+author: "Kornelia Ćwik"
+date: "25 04 2020"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(sf)
+library(readxl)
+library(spData)
+library(ggplot2)
+library(dplyr)
+```
+
+## Zad. 1
+
+Utworzenie punktu znajdującego się na terenie Polski.
+
+```{r zad1}
+lnd_point = st_point(c(16.56, 52.24))           #poznań      
```

Comment:
Do przypisywania zmiennych używamy <- . Jest to dobra praktyka czytelności kodu. "=" też zadziała, ale nie jest zazwyczaj uznawana za przejaw czytelnego kodu.  "=" jest wykorzystywane do argumentów wewnątrz funkcji więc warto to rozrózniać.